py-utils/src/utils/msg_bus/confluent_kafka_example.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import

```

Comment:
Please move it to py-utils/test/message_bus/

Reply:
Moved